cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -4031,6 +4033,142 @@ action.cyences_send_email_action = 1
+relation = greater than
+cron_schedule = 13 * * * *
+description = Alerts when attack is not blocked
+dispatch.earliest_time = -1h@m
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_imperva_waf` sourcetype=imperva:waf action=REQ_PASSED \
+| stats count by src category severity \
+| eval cyences_severity = if(severity="unknown","high",severity) \
+| `cs_imperva_waf_not_blocked_attacks_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_imperva_waf_not_blocked_attacks_filter
+action.cyences_notable_event_action.contributing_events =  `cs_imperva_waf` sourcetype=imperva:waf action=REQ_PASSED
+action.cyences_notable_event_action.attacker_search = | stats count by src
+action.cyences_notable_event_action.attacker_drilldown =  `cs_imperva_waf` sourcetype=imperva:waf action=REQ_PASSED src=$row.src$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Imperva WAF
+action.cyences_notable_event_action.teams = SOC
+
+# =======================
+# Imperva DAM
+# =======================
+[Imperva DAM - Logins Outside Working Hours]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 12 * * * *
+description = Alerts when someone tries to login outside working hours
+dispatch.earliest_time = -1h@m
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_imperva_dam` sourcetype=imperva:dam:internal_audit event_type="User admin logged in" \
+| `cs_outside_working_hour_definition` \
+| `cs_human_readable_time_format(_time,event_time)` \
+| `cs_user_privilege_mapping(suser)` \
+| eval cyences_severity = if(is_privileged_user=="Yes", "critical", "high") \
+| table event_time suser src_ip cyences_severity \
+| `cs_imperva_dam_logins_outside_working_hours_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_imperva_dam_logins_outside_working_hours_filter
+action.cyences_notable_event_action.contributing_events =  `cs_imperva_dam` sourcetype=imperva:dam:internal_audit event_type="User admin logged in"
+action.cyences_notable_event_action.attacker_search = | stats count by src_ip
+action.cyences_notable_event_action.attacker_drilldown =  `cs_imperva_dam` sourcetype=imperva:dam:internal_audit event_type="User admin logged in" src_ip=$row.src_ip$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Imperva DAM
+action.cyences_notable_event_action.teams = SOC
+
+[Imperva DAM - Multiple Failed Logins]
```

Comment:
Filter macro name does not match with the saved search name - cs_imperva_dam_failed_logins_by_user_filter

=> cs_imperva_dam_multiple_failed_logins_filter

Check this across all alerts.